components/higher-order/with-focus-outside/test/index.js

```diff
@@ -12,9 +12,12 @@ import { Component } from '@wordpress/element';
  * Internal dependencies
  */
 import withFocusOutside from '../';
+import ReactDOM from 'react-dom';
 
 jest.useFakeTimers();
```

Comment:
Aside: we have fake timers enabled by default ...

Reply:
That was already in this test, so likely can just be an additional improvement to the original test?

Reply:
If you can remove at the same time, it would be awesome. It should work 😓

Reply:
ya looks like tests pass fine with it removed.